src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.prod {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.prod__img {
+  width: 100%;
```

Comment:
'width' must be not 100% because if parent element will be bigger, image will broke.

Reply:
Agree, set fixed width for the image